homeassistant/components/met_eireann/weather.py

```diff
@@ -108,7 +109,7 @@ def name(self):
 
         return f"{DEFAULT_NAME}{name_appendix}"
 
-    @property
```

Comment:
This one can be done in the constructor, needs a separate PR

Reply:
#100335